relay/http.go

```diff
@@ -136,6 +139,23 @@ func (h *HTTP) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 	queryParams := r.URL.Query()
 
+	// Don't pass through non create queries
+	if reqPath == "/query" {
+		// First check if there are multiple queries being passed
+		queries := strings.Split(queryParams["q"][0], ";")
+		if len(queries) > 1 {
+			// If there are drop any queries other than the first
+			queryParams["q"][0] = queries[0]
```

Comment:
silently dropping the query string, then returning a success is definitely the wrong thing to do. multiple statements should either be rejected, or processed individually.